packages/kamino-sdk/src/Kamino.ts

```diff
@@ -2616,7 +2616,14 @@ export class Kamino {
       }
     }
 
-    return { prerequisiteIxs: collectFeesAndRewardsIxns, withdrawIx };
+    let res: WithdrawShares = { prerequisiteIxs: collectFeesAndRewardsIxns, withdrawIx };
+    // if we withdraw everything also close the shares ATA
+    const sharesAtaBalance = await this.getTokenAccountBalance(sharesAta);
```

Comment:
can you not get this as an argument pls instead of making another rpc account? the ui already has this information